.github/workflows/main.yaml

```diff
@@ -256,7 +258,7 @@ jobs:
           rm -rf awscliv2.zip aws/
 
       - name: Provisioning cluster tests
-        if: ${{ !cancelled() && steps.prepare-rancher.outcome == 'success' && contains(inputs.tests_to_run, 'p0_provisioning') }}
+        if: ${{ !cancelled() && (inputs.RANCHER_HOSTNAME != 'hostname' || steps.prepare-rancher.outcome == 'success') && contains(inputs.tests_to_run, 'p0_provisioning') }}
```

Comment:
I don't get the extended condition with `inputs.RANCHER_HOSTNAME != 'hostname'`, I don't see such a input defined anywhere.